src/Fixtures/en/frisbeeDrills.en.js

```diff
@@ -5424,7 +5453,8 @@ const aussieDrill = {
   author: 'Head of States',
   title: 'Aussie games',
   image: 'https://zupimages.net/up/20/19/78g2.png',
-  description: 'You can run with the disc',
+  description:
+    "You can run with the disc\n\nIn game: This playful exercise allows to work on the defenders' communication. This is essential in order to realize switches on the field and/or an efficient zone defense. In attack, this exercise allows you to work on the detection of the players who are not marked out.",
```

Comment:
"In attack" => "On offense"

"this exercise allows" => "this exercise enables" (also in other drill descriptions)